xml/System.Text.Json/JsonSerializerDefaults.xml

```diff
@@ -34,13 +34,16 @@
       </ReturnValue>
       <MemberValue>0</MemberValue>
       <Docs>
-        <summary>Specifies that general-purpose values should be used. These are the same settings applied if a <see cref="T:System.Text.Json.JsonSerializerDefaults" /> isn't specified.</summary>
+        <summary>General-purpose option values. These are the same settings that are applied if a <see cref="T:System.Text.Json.JsonSerializerDefaults" /> member isn't specified.</summary>
         <remarks>
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
 
-This option implies that property names are treated as case-sensitive and that "PascalCase" name formatting should be employed.
+This member implies that:
+
+- Property names are treated as case-sensitive.
+- "PascalCase" name formatting should be employed.
```

Comment:
Not sure that we have to mention this. If we do, what about all the other options e.g. `ReadCommentHandling`, `DictionaryKeyPolicy` etc.

If the intention is to contrast with `Web` options, maybe we should add that quoted numbers (JSON strings) are not allowed.

Reply:
I was mainly just reformatting the existing text. But it seems like it would be helpful to list out all the options it prescribes?

Reply:
@tdykstra What do you think?

Reply:
I think we should link to here: https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializeroptions?view=net-5.0#properties since this ideally should list what the `default` value for each property is.

Reply:
Agree with @layomia proposal, in which case explicitly mentioning property naming defaults isn't necessary here.

Reply:
Interestingly, remarks for enum members seem to be bogus (ignored). I'll double check that that behavior is by design.

https://review.docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializerdefaults?view=net-5.0&branch=pr-en-us-5007